src/convertToObject.js

```diff
@@ -12,7 +12,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  const arr = sourceString.split(';').map(elem => elem.trim());
+
+  const newArr = arr.filter(char => Boolean(char));
```

Comment:
```suggestion
  const newArr = arr.filter(Boolean);
```
you can write even shorter